x-pack/docs/en/security/securing-communications/tls-versions-jdk.asciidoc

```diff
@@ -0,0 +1,119 @@
+  See <<jdk-enable-tls-protocol>>
+
+`TLSv1`::
+  TLS v1.0 is supported on all JDKs.
+  Some newer JDKs, including the JDK bundled with {es} disable TLS v1.0 by default
+  See <<jdk-enable-tls-protocol>>
+
+`TLSv1.1`::
+  TLS v1.1 is supported on all JDKs.
+  Some newer JDKs, including the JDK bundled with {es} disable TLS v1.1 by default
+  See <<jdk-enable-tls-protocol>>
+
+`TLSv1.2`::
+  TLS v1.2 is supported on all JDKs.
+  It is enabled by default on all JDKs that are supported by {es}, including the bundled JDK.
+
+`TLSv1.3`::
+  TLS v1.3 is supported on all recent JDKs. This includes the latest release of Java 8
+  (any release newer than 8u261) as well as all versions of Java 11 or later.
+  It is supported and enabled by default on the {es} bundled JDK.
+
+NOTE: The information above applies when running on the default JSSE provider in the JDK.
+      JVMs that are configured to use a <<fips-140-compliance, FIPS 140-2>> security
+      provider may have a custom TLS implementation and the supported TLS protocol versions
+      may differ from the list above.
+      +
+      Please check the release notes for your security provider.
+
+[[jdk-enable-tls-protocol]]
+==== Enabling additional SSL/TLS versions on your JDK
+
+The set of supported SSL/TLS versions for a JDK is controlled by a configuration file.
+It is possible to override values that are set in this configuration file in order to
+enable legacy SSL or TLS versions.
+
+The JDK configuration file lists the SSL/TLS algorithms that are disabled in that JDK.
+Removing a TLS version from that list will allow it to be used on the JDK.
+To do this, you should follow these steps:
+
+1. Locate the configuration file for your JDK
+2. Copy the `jdk.tls.disabledAlgorithms` setting from that file, and add it to a custom 
+   configuration file within the {es} configuration directory
+3. Modify the value of the `jdk.tls.disabledAlgorithms` setting so that the TLS version
+   that you wish to use is no longer listed
+4. Configure {es} to pass a custom system property to the JDK so that your custom
+   configuration file is used.
+
+===== Locate the configuration file for your JDK
+
+For the {es} **bundled JDK**, the configuration file is in a sub directory of the {es} home directory (`$ES_HOME`).
+
+* On Linux the configuration file is `$ES_HOME/jdk/conf/security/java.security` 
+* On Windows the configuration file is `$ES_HOME/jdk/conf/security/java.security` 
+* On macOS the configuration file is `$ES_HOME/jdk.app/Contents/Home/conf/security/java.security` 
+
+For **JDK8**, the configuration file is within the `jre/lib/security` directory of the Java installation.
+If `$JAVA_HOME` points to the home directory of the JDK that you use to run {es}, then the configuration file will be
+
+* `$JAVA_HOME/jre/lib/security/java.security`
+
+For **JDK11 or later**, the configuration file is within the `conf/security` directory of the Java installation.
```

Comment:
This does not include JDK9 and 10? They seem to match the pattern here (tested only with my local installations).

Reply:
Java 9 and 10 are EOL.
No supported version of ES is compatible with them.